### PR TITLE
Resolves #378 - Module version with the version information

### LINKED ISSUE #378: Determining the version of libopenssl
I haven't found a way to display the version of OpenSSL to which this crate is linked. It would be useful for troubleshooting.

### PR DESCRIPTION
**Pull request by Yoric**

### CODE REVIEW

**Comment by Yoric** [comment]
The failures seem to be due to an incompatibility between Rust Nightly and pre-existing code.

**Comment by sfackler** [comment]
Yeah, don't worry about the nightly builds. The appveyor failure appears legit though.

**Comment by Yoric** [comment]
Yes, I've just fixed it, I believe.

**Comment by Yoric** [comment]
@sfackler Ok, fix confirmed.

File: openssl/src/version.rs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+/// `0x000905000 == 0.9.5 dev`
+///
+/// Version 0.9.5a had an interim interpretation that is like the current one, except the patch level got the highest bit set, to keep continuity. The number was therefore 0x0090581f
+///
+/// The return value of this function can be compared to the macro to make sure that the correct version of the library has been loaded, especially when using DLLs on Windows systems.
+pub fn number() -> i64 {
+    unsafe { ffi::SSLeay() as i64 }
+}
+
+const SSLEAY_VERSION : c_int = 0;
```
Comment (sfackler) [inline openssl/src/version.rs]: These constants should go in openssl-sys as well.

**Comment by sfackler** [comment]
Looks good to me other than moving some constants, thanks!

**Comment by Yoric** [comment]
Done.

**Comment by sfackler** [comment]
Thanks!

**Pull request merged**